recipes/pdspy/run_test.sh

```diff
@@ -0,0 +1,5 @@
+export OMPI_MCA_plm=isolated
+export OMPI_MCA_btl_vader_single_copy_mechanism=none
+export OMPI_MCA_rmaps_base_oversubscribe=yes
```

Comment:
Should the above live in an activation script?

Reply:
This only seems to be an issue for the conda-forge build test system, I think related to the availability of an MPI version. So I think this is only needed for the test.

Reply:
Well, if these aren't defined for the tests (and they are needed for normal execution) then they won't be defined for users who install the package either

Reply:
I don't believe they need to be defined, in general, for users who install the package, and is just to get through the tests. I added the test script due to a build error during the below testing, but that seems to be specific to that system and not to systems in general (I have never needed them on any system I've installed on). This is following exactly what mpi4py does: https://github.com/conda-forge/mpi4py-feedstock/commit/23aef0ba1633339dc807b82fba985c034e41c9b0#diff-50e56299841cca340fa9c08824016349, which makes sense because the error happens on importing mpi4py.

Reply:
Ok if they are just for tests, than this is fine

Reply:
Ok great, thanks!